Milione/graph.gv

```diff
@@ -0,0 +1,41 @@
+# Place the cursor inside "graph" to get some refactoring options
+
+digraph {
+    # Greek letters/hypothetical links
+    omega[color="grey", label="Ω"]
+    alpha[color="grey", label="α"]
+    beta[color="grey", label="β"]
+    delta[color="grey", label="δ"]
+    delta¹[color="grey", label="δ¹"]
+    delta²[color="grey", label="δ²"]
+    delta³[color="grey", label="δ³"]
+    gamma[color="grey", label="γ"]
+    epsilon[color="grey", label="ε"]
+
+    omega -> alpha;
+    omega -> beta;
+
+    alpha -> gamma;
+    gamma -> V;
+    alpha -> delta;
+    delta -> Fr;
+    Fr -> delta²;
```

Comment:
Shouldn't `Fr` be greyed out ?

Reply:
Don't think so

<img width="654" alt="Screen Shot 2020-12-19 at 19 15 12" src="https://user-images.githubusercontent.com/26046460/102696443-9de6a780-422e-11eb-9122-b2794bf50b07.png">